srcpkgs/runc/template

```diff
@@ -11,11 +11,11 @@ go_build_tags="seccomp apparmor"
 hostmakedepends="pkg-config go-md2man"
 makedepends="libseccomp-devel"
 short_desc="CLI tool for spawning and running OCI containers"
-maintainer="Paul Knopf <pauldotknopf@gmail.com>"
```

Comment:
Is it you? If not, is the current maintainer aware? Why is is orphaned?

Reply:
The current maintainer (@pauldotknopf) hasn't been active. His last update was [2 years ago](https://github.com/void-linux/void-packages/commits?author=pauldotknopf)